README.rst

```diff
@@ -31,36 +30,29 @@ Features
 Requirements
 ------------
 
-django-timepiece is compatible with Python 2.{6,7}, Django 1.{4,5}, and
-PostgreSQL. PostgreSQL is the only offically supported database backend and,
-therefore, requires `psycopg2 <http://initd.org/psycopg/>`_. django-timepiece
-also depends on the following Django apps:
-
- * `python-dateutil <http://labix.org/python-dateutil>`_
- * `django-selectable <http://pypi.python.org/pypi/django-selectable>`_
- * `django-pagination <http://pypi.python.org/pypi/django-pagination>`_
- * `django-compressor <https://github.com/jezdez/django_compressor>`_
- * `django-bootstrap-toolkit <https://github.com/dyve/django-bootstrap-toolkit>`_
+django-timepiece is compatible with Django 1.6 (on Python 2.7 and Python 3.3),
+and Django 1.7 (on Python 2.7 and Python 3.4). PostgreSQL is the only
+officially supported backend. For a full list of required libraries, see
+the `requirements`_.
 
 We actively support desktop versions of Chrome and Firefox, as well as common
 mobile platforms. We do not support most versions of Internet Explorer. We
 welcome pull requests to fix bugs on unsupported browsers.
 
-django-timepiece uses Sphinx and RST for documentation. You can use Sphinx to
-build the documentation:
+Documentation
+-------------
 
- * `docutils <http://docutils.sourceforge.net/>`_
- * `Sphinx <http://sphinx.pocoo.org/>`_
+Documentation is hosted on `Read The Docs`_.
 
-A makefile is included with the documentation so you can run `make html` in the
-`doc/` directory to build the documentation.
+To build the documentation locally, install the `documentation requirements`_.
+A makefile is included with the documentation so you can run `make html` in
+the `doc/` directory to build docs as HTML.
 
 Installation
 ------------
 
-#. django-timepiece is available on `PyPI
-   <http://pypi.python.org/pypi/django-timepiece>`_, so the easiest way to
-   install it is to use `pip <http://pip.openplans.org/>`_::
+#. django-timepiece is available on `PyPI`_, so the easiest way to
+   install it and its dependencies is to use `pip <http://pip.openplans.org/>`_::

```

Comment:
Because it fit comfortably on the line, and wasn't re-used, so it didn't bother me :smile_cat: But I'm happy to make a change for consistency.